__snapshots__/schema-validator.test.js

```diff
@@ -0,0 +1,19 @@
+exports['schema validator env with invalid env should fail 1'] = `
+
+	version: "version" must be one of [1.0.0]
```

Comment:
I think this should be "schemaVersion", not simply "version". "Version" might confuse customers that it is meant to hold the version of the backend configuration file.